python/agents/camel/camel/camel_agent/camel_agent.py

```
@@ -0,0 +1,587 @@
+  loop_agent: LoopAgent
+
+  model_config = {"arbitrary_types_allowed": True}
+
+  def __init__(
+      self,
+      name: str,
+      model: str | BaseLlm = "gemini-2.5-pro",
+      description: str = "",
+      instruction: str = "",
+      tools: Optional[list[Tool]] = None,
+      security_policy_engine: SecurityPolicyEngine = security_policy.NoSecurityPolicyEngine(),
```

Ruff (0.12.2)

Do not perform function call `security_policy.NoSecurityPolicyEngine` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)